scripts/timelapse-colorizer-data/convert_nucmorph_data.py

```diff
@@ -189,7 +189,9 @@ def make_dataset(output_dir="./data/", dataset="baby_bear", do_frames=True, scal
     for feature in FEATURE_COLUMNS:
         (scale_factor, label, unit) = get_plot_labels_for_metric(feature)
         feature_labels.append(label.capitalize())
-        feature_metadata.append({"units": formatted_units[unit]})
+        unit = formatted_units.get(unit)
+        if unit:
+            feature_metadata.append({"units": unit})
```

Comment:
I guess adding `"": None,` to the lookup table wouldn't hurt too.   Can there be bugs if a unit gets skipped in this way?